### PR TITLE
Acknowledge MultiMC Quilt support

### PR DESCRIPTION
**Pull request by arthomnix**
MultiMC has recently added support for Quilt (MultiMC/Launcher#3760), so this PR moves MultiMC from 'Awaiting Support' to 'Launchers' and adds installation instructions for MultiMC.

### CODE REVIEW

**Review by wafflecoffee — APPROVED** [review]

File: src/install/multimc.md [hunk 1]
```diff
@@ -6,18 +6,31 @@ permalink: /install/multimc/
...
+   your instance
+8. Click **OK** to create a new vanilla instance
+9. Right click on the new instance and select **Edit Instance**, or click the instance once and select
+   **Edit Instance** from the panel on the right
+10. Make sure **Version** is selected on the left, and press the **Install Quilt** button on the menu
+    on the right of the window
+11. Select the Quilt Loader version you'd like to use and press **OK**
+12. Set up the rest of your instance as normal
+
+That's all there is to it! Once you've followed these steps, your instance should be ready to go &mdash; just add your
```
Comment (gdude2002) [inline src/install/multimc.md:32]: No need, `--` will work for Markdown files

**Review by gdude2002 — CHANGES_REQUESTED** [review]
I think this is mostly OK, but I'd prefer it to follow the format of the PolyMC guide - in fact, given that the steps are identical for both, they really _should_ be identical other than the links and names, imo.

File: src/install/multimc.md [hunk 2]
```diff
@@ -6,18 +6,29 @@ permalink: /install/multimc/
...
-[submitted a pull request](https://github.com/MultiMC/Launcher/pull/3760) that adds Quilt support. If you're 
-interested in seeing Quilt support in MultiMC, feel free to take a look at that PR.
-
-# Manual Installation
-
-While it is technically possible to add Quilt to any MultiMC instance manually, it's a somewhat technical process that
-involves writing your own JSON component definition. For that reason, we're not publishing instructions on how to do 
-that here &mdash; however, we are working on automatically generating MultiMC instances you can import to easily get 
-set up with Quilt.
+{% admonition Development version required/danger %}
```
Comment (gdude2002) [inline src/install/multimc.md:9]: Without the header title for consistency - just `/danger` is correct. That's a feature we never ended up using, haha - you can stick it in a h2 in the admonition itself instead.

**Review by gdude2002 — APPROVED** [review]
You could've just used a markdown header, but this is alright, haha - thanks for the contribution!

**Pull request merged**